src/components/organisms/PublicFooter.vue

```diff
@@ -0,0 +1,76 @@
+<script setup>
+import { computed } from 'vue'
+import { APP_TITLE } from '~/constants'
+import DcLocalePicker from '../molecules/DcLocalePicker.vue';
+import { useI18n } from 'vue-i18n';
+
+const { t } = useI18n()
+
+const menu = computed(() => [
+  {
+    title: t('button.about'),
+    href: '/#about',
+  },
+  {
+    title: t('button.contact_us'),
+    href: '/#contact-us'
+  }
+])
+</script>
+
+<template>
+  <QFooter elevated class="public-footer bg-dark">
+    <div class="public-footer__wrapper">
+      <QToolbar class="row justify-between q-pa-md">
+        <div class="column row-md q-gutter-y-lg q-gutter-x-lg">
+          <RouterLink
+            class="public-footer__link text-subtitle1"
+            v-for="(item, index) in menu"
+            :key="index"
+            :to="item.href"
+          >
+            {{ item.title }}
+          </RouterLink>
+        </div>
+
+        <div>
+          <DcLocalePicker />
+        </div>
+      </QToolbar>
+      <QSeparator dark />
+      <QToolbar>
+        <QToolbarTitle class="public-footer__toolbar row q-gutter-x-md items-center q-pa-md">
+          <QAvatar size="30px">
+            <img src="/favicon.svg">
+          </QAvatar>
+          <div class="text-h6 text-weight-bold flex">{{APP_TITLE}}<span class="text-caption "> &copy; {{ new Date().getFullYear() }}</span></div>
```

Comment:
It might not be a good practice to have strict javascript logic within the template. I could suggest creating a computed prop for the FullYear.

Reply:
Ok, moved the logic to an `actualYear` computed prop